static/app/views/issueDetails/groupReplays/groupReplaysPlayer.tsx

```diff
@@ -59,20 +61,22 @@ export default function GroupReplaysPlayer({
 
         return (
           <PlayerContainer data-test-id="player-container">
-            {replay?.hasProcessingErrors() ? (
-              <ReplayProcessingError processingErrors={replay.processingErrors()} />
```

Comment:
Where is this handled now?

Reply:
oh, good call out. i meant to comment

It's inside the `<ReplayLoaderState>` which is already in this file.

The last place that still needs a LoaderState added is the main replay details page; so that's not in this PR yet